src/components/Filter/FilterGroup.js

```diff
@@ -33,6 +33,22 @@ const FilterGroup = ({
         collapseFilterGroup(groupKey, !isCollapsed);
       }
     };
+
+    /* Sort filterItems alphabetically
+    ** Taken from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
+    */ 
+    if (groupKey !== 'grade') {
+      filterItems.sort((a, b) => {
```

Comment:
Kan jeg foreslå:
```
filterItems.sort((a, b) => a.props.tagName.localeCompare(b.props.tagName));
```
?

Reply:
Jeg ser også at array med filteritems kan være `null` (som gjør at sort kan feile). Hva med å chain'e map'en med
```
.filter(item => !!item) // filter out null-values
```
for å få vekk disse?

Reply:
Og jeg ville kanskje flyttet sort-setningen opp rett under der filterItems blir laget...